docs/system_overview/genquery.md

```diff
@@ -553,3 +551,281 @@ Find all data objects with replicas on resource `replResc` which are not marked
 ```
 SELECT COLL_NAME, DATA_NAME WHERE DATA_RESC_NAME = 'replResc' and DATA_REPL_STATUS != '1'
 ```
+
+## GenQuery2
+
+GenQuery2 is an experimental parser designed to give users more control over their queries.
+
+This section will cover the differences between GenQuery1 and GenQuery2.
+
+!!! Note
+    GenQuery2 is experimental and may change based on user feedback. However, we encourage its use and welcome all feedback.
+
+### Attribute Names
+
+GenQuery2 supports a subset of the GenQuery1 attributes. Some of the attributes have been renamed to better express their intent.
+
+Below is a listing of each attribute that was renamed in GenQuery2.
+
+| GenQuery1 Attribute | Equivalent GenQuery2 Attribute |
+|---|---|
+| COLL_ACCESS_NAME | COLL_ACCESS_PERM_NAME |
+| COLL_ACCESS_TYPE | COLL_ACCESS_PERM_ID |
+| DATA_ACCESS_NAME | DATA_ACCESS_PERM_NAME |
+| DATA_ACCESS_TYPE | DATA_ACCESS_PERM_ID |
+| RESC_LOC | RESC_HOSTNAME |
+| RULE_EXEC_ADDRESS | DELAY_RULE_EXE_ADDRESS |
+| RULE_EXEC_ESTIMATED_EXE_TIME | DELAY_RULE_ESTIMATED_EXE_TIME |
+| RULE_EXEC_FREQUENCY | DELAY_RULE_EXE_FREQUENCY |
+| RULE_EXEC_ID | DELAY_RULE_ID |
+| RULE_EXEC_LAST_EXE_TIME | DELAY_RULE_LAST_EXE_TIME |
+| RULE_EXEC_NAME | DELAY_RULE_NAME |
+| RULE_EXEC_NOTIFICATION_ADDR | DELAY_RULE_NOTIFICATION_ADDR |
+| RULE_EXEC_PRIORITY | DELAY_RULE_PRIORITY |
+| RULE_EXEC_REI_FILE_PATH | DELAY_RULE_REI_FILE_PATH |
+| RULE_EXEC_STATUS | DELAY_RULE_STATUS |
+| RULE_EXEC_TIME | DELAY_RULE_EXE_TIME |
+| RULE_EXEC_USER_NAME | DELAY_RULE_USER_NAME |
+
+As mentioned earlier, GenQuery2 exposes a subset of the GenQuery1 attributes. That means, many of the attributes were dropped from GenQuery2. Notable attributes include the following:
+
+- `COLL_OWNER_NAME`
+- `COLL_OWNER_ZONE`
+- `DATA_OWNER_NAME`
+- `DATA_OWNER_ZONE`
+
+Logic relying on these attributes will need to be updated to use the permission attributes.
+
+To get the full listing of attributes, use `iquery -c`. The listing can also be retrieved programmatically via the `rc_genquery2` API endpoint.
+
+!!! Note
+    The results always reflect the attributes supported by the GenQuey2 implementation of the iRODS Catalog Provider.
```

Comment:
```suggestion
    The results always reflect the attributes supported by the GenQuery2 implementation of the iRODS Catalog Provider.
```

I'm a little confused by this sentence, specifically the "of the iRODS Catalog Provider" clause. Can you explain what that means?

Reply:
The GenQuery2 API goes to the catalog provider to build the list of supported attributes/column-names rather than the directly connected [consumer] server.

Reply:
Adjusting wording.

Reply:
Done.